### DISCUSSION TITLE
Add some missing details to main menu

### DISCUSSION THREAD

**Discussion by Biokax** [ppy/osu#18276] [Ideas] [answered]
Some details are missing on lazer in the main menu, I really think people care about it, for example I don't know how many maps I have on lazer :

- "You have XXXXX beatmaps"
- Osu player level and accuracy
- Osu player PP and rank

*Stable*
![image](https://user-images.githubusercontent.com/50368878/168467526-7204e629-f614-4825-b0df-4d800570e471.png)

*Lazer*
![image](https://user-images.githubusercontent.com/50368878/168467534-47337708-3719-42df-9777-3fef5d033c68.png)

Its great to have minimalist main menu, but having more detailed interface when clicking on play (like stable) can be a good QOL content

**Comment by peppy** [comment, marked as answer]
- Click your avatar / username to get player stats.
- Beatmap count will not be shown at the main menu. It will be in settings and at song select eventually.

**Reply by Biokax** [reply]
showing beatmaps count in settings is not really intuitive, however in song select it can be cool if its well integrated in the UX

**Reply by 1llaN** [reply]
I think it would be nice if the global rank and PP were always displayed at the top next to the avatar, rather than after completing a map, as they are now. At least as an option.
